lib/hunter/commands/hunt.rb

```diff
@@ -158,7 +158,8 @@ def process_packet(data:)
           presets: {
             label: data["label"],
             prefix: data["prefix"]
-          }
+          },
+          filepath: File.join(Config.node_buffer, data["hostid"])
```

Comment:
Given a `hostid` of `1234abcd`, this will result in a `Node` being saved at `$node_buffer/1234abcd`. I think you may be better off storing a node's parent `NodeList` object as an instance variable on `Node` instead of `filepath`, and having a getter method for `filepath` which evaluates the filepath like this:

```ruby
class Node
  def filepath
    File.join(node_list.dir, "#{id}.yaml")
  end
end
```

The new behaviour for moving a node to another list (as seen in `commands/parse`) would be:

```ruby
@buffer.delete([node])
@parsed.nodes.concat([node])
@parsed.save
```

instead of having to modify a node's filepath inplace, like the current behaviour.

Reply:
Good idea, having the filepath as a modifiable attribute felt pretty awkward but a better solution wasn't coming to me.

Reply:
It's times like this when I miss having `ActiveRecord` as an included library.